### DISCUSSION TITLE
Get build log when job fails?

### DISCUSSION THREAD

**Discussion by jchrisweaver** [concourse/concourse#8305] [Help & Support]
When a job fails, we have an on_failure hook that sends an email that the job failed.  However, I'd like to attach the build log, too.  How can I access the build log for the job that failed?  Thanks!

EDIT for clarity:  What I'm looking for would be the results of the `fly watch` command.  How can I capture that?

**Comment by marco-m** [comment]
You should be able to do this from within a resource, using the same reasoning as in https://github.com/concourse/concourse/discussions/7919#discussioncomment-2885350

**Reply by jchrisweaver** [reply]
Is there an example of doing something like this that you could point me to?  This is pretty advanced.
